templates/taxi/car_detail.html

```diff
@@ -1,12 +1,27 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% load crispy_forms_tags %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <p>Manufacturer: {{ car.manufacturer.name }}, {{ car.manufacturer.country }}</p>
   <h1>Drivers</h1>
   <hr>
   <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
+    {% if car.drivers.all %}
```

Comment:
The condition `{% if car.drivers.all %}` will always evaluate to `True` because `car.drivers.all` is a queryset object. Consider using `{% if car.drivers.exists %}` to check if there are any drivers associated with the car.